config/jobs/kubernetes/minikube/minikube-presubmits.yaml

```diff
@@ -262,3 +262,33 @@ presubmits:
           limits:
             memory: 20Gi
             cpu: 7
+  - name: integration-vfkit-docker-macos-arm64
+    cluster: k8s-infra-prow-build
+    optional: true # macos machine is precious, so only run when matainers explicitly trigger with /test <job_name>
+    decorate: true
+    path_alias: "k8s.io/minikube"
+    always_run: true
+    labels:
```

Comment:
I think it is better to run in a arm64 Pod, (add arm64 node selector) so when it installs golang it installs arm64 one...that way it GOOS will return arm64 for cross build script